serialization.md

```diff
@@ -422,14 +362,13 @@ bin/my-app-binary
 bin/my-app-tools
 ```
 
-The TarSum checksum for the archive file is then computed and placed in the
-JSON metadata along with the execution parameters.
+The TarSum checksum for the archive file is then computed and placed in the JSON metadata along with the execution parameters.
```

Comment:
ah nice. That should probably be removed from the distribution spec too. :-)

On Fri, Apr 8, 2016 at 1:32 PM, Stephen Day <notifications@github.com>
wrote:

> In serialization.md
> <https://github.com/opencontainers/image-spec/pull/12#discussion_r59069877>
> :
>
> > @@ -422,14 +362,13 @@ bin/my-app-binary
> >  bin/my-app-tools
> >  ```
> >
> > -The TarSum checksum for the archive file is then computed and placed in the
> > -JSON metadata along with the execution parameters.
> > +The TarSum checksum for the archive file is then computed and placed in the JSON metadata along with the execution parameters.
>
> I'm not sure if this is in scope for this PR, but a TarSum isn't used
> here. This should probably just say digest.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/12/files/c146488d55d702f2c7103da26ac5a20dfc5f34c1#r59069877>
>